lib/rouge/lexers/python.rb

```diff
@@ -7,7 +7,7 @@ class Python < RegexLexer
       desc "The Python programming language (python.org)"
       tag 'python'
       aliases 'py'
-      filenames '*.py', '*.pyw', '*.sc', 'SConstruct', 'SConscript', '*.tac'
+      filenames '*.py', '*.pyw', 'SConstruct', 'SConscript', '*.tac'
```

Comment:
i have no idea but @jneen was ok with removing it in the convo above. i do not have much of an interest in discussing the other points you raised, but if you decide something let me know; i am happy to make the rest of the changes once this point is resolved. also, frankly, it is very frustrating to have this sit in a semi-approved state for over a year and then suddenly have the goalposts moved for no clear reason.

Reply:
@brianlheim Sorry, didn't see she'd been fine with that, I was trying to get through as many changesets as I could and missed it. 

To be clear I'm not a maintainer - I've discussed with one of the maintainers what I can do to help out in getting things to a state where we can get them merged exactly because I too am frustrated as someone relying on Rouge, and want to help get this moved along. 

Since @jneen has agreed it's fine for you to use the .sc extension, that of course stands.

Since you're ok going ahead with the other changes, once you do that I'll add this changeset to the list I think are ready for @dblessing to merge.

Reply:
Removing `.sc` would be a breaking change and should be avoided. We can write a disambiguation that distinguishes between Python and SuperCollider. I've submitted this as [a PR](https://github.com/brianlheim/rouge/pull/1) to @brianlheim's branch.